src/edu/ucsd/library/xdre/tab/FilesChecker.java

```diff
@@ -154,6 +172,10 @@ public boolean filesValidate ()
 			// Build the command for file validation
 			List<String> cmd = new ArrayList<>();
 			cmd.add( command );
+			if (src.length() > MAX_FILESIZE) {
```

Comment:
@escowles Thanks. I've removed the restriction to always enable it. See https://github.com/ucsdlib/damsmanager/commit/2fbfa24a37a9702469ba02670a4d9caf4dcbbb43.